src/Esfa.Vacancy.Register.Infrastructure/Repositories/VacancyRepository.cs

```diff
@@ -47,7 +48,33 @@ public VacancyRepository(IProvideSettings provideSettings, ILog logger)
                 await sqlConn.OpenAsync();
                 var results =
                     await sqlConn.QueryAsync<DomainEntities.Vacancy, DomainEntities.Address, DomainEntities.Vacancy>(
-                        GetLiveVacancyByReferenceNumberSqlSproc,
+                        GetLiveApprenticeshipVacancyByReferenceNumberSqlSproc,
+                        param: parameters,
+                        map: (v, a) => { v.Location = a; return v; },
+                        splitOn: "AddressLine1",
+                        commandType: CommandType.StoredProcedure);
+
+                vacancy = results.FirstOrDefault();
+            }
+
+            return vacancy;
+        }
```

Comment:
What's the benefit of having both apprenticeship and traineeship funcs in this repo? As they are using different sprocs anyway they are using different db objects.